src/command.js

```diff
@@ -43,6 +43,11 @@ export function matchImageSnapshotCommand(defaultOptions) {
           diffOutputPath,
         }) => {
           if (!pass && !added && !updated) {
+            if ((commandOptions.retryCounter || 0) > 0) {
+              cy.wait(100);
```

Comment:
this 100 ms is fine as a default but it should be one of the options and be configurable

Reply:
done